### PR TITLE
qtermwidget: Add recipe (wip) no merge

### PR DESCRIPTION
**Pull request by khallebal**
It's not building with haikuports, i keep getting the following.

```sh
waiting for build package qtermwidget_x86-0.14.1-1 to be activated
```
I can't find what's wrong with the recipe, so i thought more eyes would help to see the error.

### CODE REVIEW

File: dev-qt/qtermwidget/qtermwidget-0.14.1.recipe [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. $cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE=Release \
```
Comment (diversys) [inline dev-qt/qtermwidget/qtermwidget-0.14.1.recipe]: Superfluous "\" at the end.

**Comment by diversys** [comment]
Have you forgotten to include qtermwidget.h changes into your patchset? It doesn't compile over here.

File: dev-qt/qtermwidget/patches/qtermwidget-0.14.1.patchset [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+diff --git a/lib/qtermwidget.cpp b/lib/qtermwidget.cpp
+index 5c3e56f..156a212 100644
+--- a/lib/qtermwidget.cpp
++++ b/lib/qtermwidget.cpp
+@@ -33,6 +33,12 @@
+ #include "SearchBar.h"
+ #include "qtermwidget.h"
+ 
++#ifdef Q_OS_MACOS
```
Comment (extrowerk) [inline dev-qt/qtermwidget/patches/qtermwidget-0.14.1.patchset]: Macos?

**Comment by extrowerk** [comment]
Please cleanup the patchset, it changes some unrelated stuff currently.

File: dev-qt/qtermwidget/qtermwidget-0.14.1.recipe (hunk 1 above)
Comment (extrowerk) [reply dev-qt/qtermwidget/qtermwidget-0.14.1.recipe]: And why os there an \ at the end pf line 59?

File: dev-qt/qtermwidget/patches/qtermwidget-0.14.1.patchset [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+ 
+ 
+-#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__DragonFly__)
++#if defined(__FreeBSD__) || defined(__DragonFly__)
+ #define HAVE_LOGIN
+ #define HAVE_LIBUTIL_H
+ #endif
+ 
+-#if defined(__OpenBSD__)
++#if defined(__OpenBSD__) || defined(__NetBSD__)
```
[the remark this answers is not in the document]
Comment (khallebal) [reply dev-qt/qtermwidget/patches/qtermwidget-0.14.1.patchset]: @extrowerk  I did not patch those lines at all,  i have no idea how they got included in my patch, anyway i've  extracted a new one.

[the remark this answers is not in the document]
Comment (khallebal) [reply dev-qt/qtermwidget/patches/qtermwidget-0.14.1.patchset]: @diversys i did not patch the qtermwidget.h  i didn't need to, my only problem is the recipe that's refusing to work, it did compile fine manually over here. i'm gonna push  the modified recipe which is still not working and a new patchset.

**Comment by khallebal** [comment]
closing, history got a bit messed up

**Pull request closed without merging**